modules/persistent-storage-csi-migration-vsphere.adoc

```diff
@@ -7,16 +7,14 @@
 = vSphere automatic migration
 
 == New installations of {product-title}
-For new installations of {product-title} 4.13, or later, automatic migration is enabled by default.
+For new installations of {product-title} 4.14, or later, automatic migration is enabled by default.
 
 == Upgrading {product-title}
-When upgrading from {product-title} 4.12, or earlier, to 4.13, automatic CSI migration for vSphere only occurs if you opt in.
+If you have vSphere in-tree persistent volumes (PVs) and want to upgrade from {product-title} 4.12 or 4.13 to 4.14, then you must upgrade to vSphere 7.0 Update 3L or 8.0 Update 2, or later. After upgrading vSphere, your {product-title} upgrade can occur and automatic migration is enabled by default.
```

Comment:
>  vSphere in-tree persistent volumes (PVs)

This is not completely accurate and should be reworded on the lines of - "If you are using in-tree vSphere volumes".

Reply:
> > vSphere in-tree persistent volumes (PVs)
> 
> This is not completely accurate and should be reworded on the lines of - "If you are using in-tree vSphere volumes".

+1 and if CSI migration is NOT already enabled



> I would also avoid using the word `MUST` when we provide a workaround right below. We should say something like `it is recommended to upgrade`. Also lets be explicit and mention that both vCenter and ESXI host should be upgraded.

Agree, it's not a must since there is a way to bypass it. It is strongly recommended and upgrades will be blocked unless the vsphere version are running the correct version or an admin provides the admin ack.
+1 on mentioning vcenter and ESXi hosts

Reply:
Fixed.